### PR TITLE
[DEV APPROVED] Add sfgb logo and link to footer

### PR DESCRIPTION
**Pull request by willhall88**
https://moneyadviceservice.tpondemand.com/entity/9970-updates-to-blog-footer

**Summary**
Currently this PR addresses changes to the Blog footer. Adding the SFGB logo, updating the background-colour and slightly changing some translations. There will be some styles that come through from Yeast in this PR - moneyadviceservice/yeast#47
I've added the background colour in Yeast as it looks like this will be used in publify as well as here in frontend. Once the Yeast PR is approved there will be one more commit to this PR to update yeast to the new version

These changes need to be rolled out on 2nd January and because of the deployment freeze over the christmas period all the changes will be worked on and ideally rolled out together, tested on staging and be the first deployment on 2nd january.

### CODE REVIEW

File: app/assets/stylesheets/layouts/_footer.scss [hunk 1]
```diff
@@ -190,33 +190,3 @@
...
-
-  @include respond-to($mq-l) {
-    @include column(8);
-    text-align: right;
-    float: right;
-    padding-bottom: 0;
-  }
-}
-
-.l-footer-secondary {
```
Comment (willhall88) [inline app/assets/stylesheets/layouts/_footer.scss]: This was actually an unnecessary style as it was being called and overridden 3 times. The colour comes through from yeast.. which now has the change to make the colour blue.

File: app/assets/stylesheets/layouts/_footer.scss [hunk 2]
```diff
@@ -190,33 +190,3 @@
   max-width: 288px;
   height: 45px;
 }
-
-.l-footer-secondary__list--left {
```
Comment (willhall88) [inline app/assets/stylesheets/layouts/_footer.scss]: these styles I have moved into the footer-secondary stylesheet

File: app/assets/stylesheets/components/footer_secondary/_footer_secondary.scss [hunk 3]
```diff
@@ -1,4 +1,40 @@
+.footer-secondary__inner {
+  @include respond-to($mq-l) {
+    display: -ms-flex;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+}
+
+.l-footer-secondary__list--left {
```
Comment (willhall88) [inline app/assets/stylesheets/components/footer_secondary/_footer_secondary.scss]: these styles I have moved from the footer stylesheet

**Comment by rhianwatkins1** [comment]
I've never worked on publify, could you please share a link to it

**Comment by willhall88** [comment]
@rhianwatkins1 
https://www.moneyadviceservice.org.uk/blog/

**Pull request merged**